### ISSUE TITLE
Revise term quest: patient discharge

### ISSUE THREAD

**Issue by dillerm** [ufbmi/OMRSE#113]
Proposed definition: a deontic document act that concludes an inpatient encounter.

Proposed parent class: deontic document act

Reason for revision: This term currently lacks a definition in OMRSE. This revision adds the definition to the term's home ontology. The definition has been updated to be consistent with the current version of d-acts.

**Comment by CDowland** [comment]
Here are some other annotations that were added in PCORnetCDM.owl, rather than the home ontology. Perhaps these should be added as well:

Term editor: Amanda Hicks

SubClassOf axiom: inverse (before) some 'admission process'
SubClassOf axiom: ends some 'inpatient encounter'

There is another 'SubClassOf' axiom, but it uses 'has active participant,' which seems to have been deprecated. I looked up 'has active participant' on Ontobee, and saw that it has an editor note saying it can be replaced by 'has agent.' However, 'has agent' is also deprecated. Perhaps replacing that with 'has participant' would suffice, even if it is not as specific as the original object property that was used. Here is the axiom, as it currently appears in the PCOR file:
(has_specified_output some 
    (document
     and ('has part' some 'objective specification')
     and ('has part' some 'action specification')
     and ('has part' some 'plan specification')))
 and ('has active participant' some 
    ('Homo sapiens'
     and ('bearer of' some 'health care provider role')))
